.tests/tests.py

```diff
@@ -8,11 +8,11 @@
 
 url = "http://127.0.0.1:8000"
 
-''' REST API CRUD FUNCTIONALITY  TESTS '''
+# REST API CRUD FUNCTIONALITY  TESTS
 
-''' MENU TESTS '''
+# MENU TESTS
 
-MenuID = 0  # global variable
+MenuID = 0
```

Comment:
can we avoid this global variable?

Reply:
pep8 recommaneds snake_case https://www.python.org/dev/peps/pep-0008/#function-and-variable-names